src/EditorFeatures/CSharp/Formatting/Indentation/CSharpIndentationService.Indenter.cs

```diff
@@ -35,7 +35,7 @@ internal class Indenter : AbstractIndenter
             {
             }
 
-            protected override IndentationResult? GetDesiredIndentationWorker(
+            protected override IndentationResult GetDesiredIndentationWorker(
```

Comment:
none of these could fail.  So returning 'null' was never possible/necessary.

Reply:
The type system is the assertion.  It literally can't be null now because we don't allow null to even be returned in the first place.  If someone attempted to do that, it would not compile.